PPPwn/run.sh

```diff
@@ -13,15 +13,11 @@ SHUTDOWN=true
 USBETHERNET=false
 
 # name of stage1 BIN"
-STAGE1="stage1"
+STAGE1="stage1_${FIRMWARE_VERSION}"
 
 # name of stage2 BIN"
-STAGE2="stage2"
+STAGE2="stage2_${FIRMWARE_VERSION}"
```

Comment:
Add VAR STAGE2 : stage1 BIN filename without ext
Helper to easy change stage2 filename